articles/azure/azs-how-create-S2S-VPN-ps.md

```diff
@@ -0,0 +1,188 @@
+
+## Create resource group
+Write-Output -InputObject "Creating resource group"
+New-AzureRmResourceGroup -Name $AzureRGName -Location $AzureLocation
+
+## Create network
+### Create subnet configurations
+Write-Output -InputObject "Creating virtual network"
+$AzureSubnetConfig = New-AzureRmVirtualNetworkSubnetConfig -Name "Default" -AddressPrefix $AzureSubnetRange
+$AzureGWSubnetConfig = New-AzureRmVirtualNetworkSubnetConfig -Name "GatewaySubnet" -AddressPrefix $AzureGWSubnetRange
+### Create virtual network
+$AzureVirtualNetwork = New-AzureRmVirtualNetwork -ResourceGroupName $AzureRGName -Location $AzureLocation -Name $AzureVNetName -AddressPrefix $AzureVNetRange -Subnet $AzureSubnetConfig,$AzureGWSubnetConfig
+### Retrieve gateway subnet config
+$AzureGWSubnetConfig = Get-AzureRmVirtualNetworkSubnetConfig -Name "GatewaySubnet" -VirtualNetwork $AzureVirtualNetwork
+
+## Create public IP address
+Write-Output -InputObject "Creating public IP address"
+$AzurePublicIP = New-AzureRmPublicIpAddress -ResourceGroupName $AzureRGName -Location $AzureLocation -AllocationMethod "Dynamic" -Name $AzurePublicIPName
+
+## Create virtual network gateway
+Write-Output -InputObject "Creating virtual network gateway"
+$AzureGatewayIPConfig = New-AzureRmVirtualNetworkGatewayIpConfig -Name "GatewayIP" -Subnet $AzureGWSubnetConfig -PublicIpAddress $AzurePublicIP
+$AzureVirtualGW = New-AzureRmVirtualNetworkGateway -ResourceGroupName $AzureRGName -Location $AzureLocation -Name $AzureVirtualGWName -IpConfigurations $AzureGatewayIPConfig -GatewayType "VPN" -VpnType "RouteBased" -GatewaySku "VpnGw1"
+
+## Create local network gateway
+Write-Output -InputObject "Creating local network gateway"
+$AzureLocalGW = New-AzureRmLocalNetworkGateway -ResourceGroupName $AzureRGName -Location $AzureLocation -Name $AzureLocalGWName  -GatewayIpAddress $AzsPublicIP.IpAddress -AddressPrefix $AzsVNetRange
+
+## Create virtual network gateway connection
+Write-Output -InputObject "Creating virtual network gateway connection"
+$AzureVirtualGWConnection = New-AzureRmVirtualNetworkGatewayConnection -ResourceGroupName $AzureRGName -Location $AzureLocation -Name $AzureGWConnectionName -VirtualNetworkGateway1 $AzureVirtualGW -LocalNetworkGateway2 $AzureLocalGW -ConnectionType IPsec -IpsecPolicies $IPsecPolicy -SharedKey $SharedKey
+
+## Retrieve public IP address of virtual network gateway
+$AzurePublicIP = Get-AzureRmPublicIpAddress -ResourceGroupName $AzureRGName -Name $AzurePublicIPName
+
+# Azure Stack
+## Reconnect to environment
+Connect-AzureRmAccount -EnvironmentName "AzureStackUser" -AccessToken $AzsAccessToken -AccountId $AzsContext.Account.Id
+
+## Set the public IP on the local network gateway
+$AzsLocalGW.GatewayIpAddress = $AzurePublicIP.IpAddress
+
+## Update the local network gateway
+Set-AzureRmLocalNetworkGateway -LocalNetworkGateway $AzsLocalGW
+</code></pre>
+
+After a short amount of time, the connection between Azure and Azure Stack should change to **Connected**.
```

Comment:
Similarly to the above, can we make sure it's clear which version of Azure/Azure Stack we're talking about

Reply:
Have changed "Azure" to "public Azure" which I think clears things up?